src/sql/workbench/browser/modelComponents/card.component.html

```diff
@@ -47,6 +47,15 @@ <h4 class="card-label">{{label}}</h4>
 				</span>
 			</div>
 		</ng-container>
+
+		<ng-container *ngIf="isImageCard">
+			<div class="card-content">
+				<div class="card-image" [style.background-image]="'url(' + imagePath + ')'" [style.background-size]="imageSize" [style.background-position]="imagePosition"></div>
```

Comment:
We already have logic in iconUtils.ts to create a CSS class for the icon - we should generally be using that (especially since a card already inherits from ComponentWithIcon which uses this logic)